frontend/src/views/web/change-password.html

```diff
@@ -4,33 +4,43 @@
+                            <span class="form-icon form-icon-dark"><i class="fa fa-key"></i></span>
+                            <label for="old_password">Old Password*</label>
+                            <div class="wrn-msg text-highlight"  ng-messages="changepwdForm.old_password.$error"  ng-if="changepwdForm.old_password.$touched || changepwdForm.$submitted">
+                                <p ng-message="minlength">password is less than 8 characters.</p>
+                                <p ng-message="required">Old password is required</p>
+                            </div>
                         </div>
 
                         <div class="input-field align-left">
-                            <input id="new_password1" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password1" focus-if>
-                            <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
-                            <label for="new_password1" data-error="wrong" data-success="right">New Password*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.new_password1">{{changepwd.wrnMsg.new_password1}}</div>
+                            <input id="new_password1" name="new_password1" type="password"  text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password1" ng-minlength="8" required >
+                            <span class="form-icon form-icon-dark"><i class="fa fa-key"></i></span>
+                            <label for="new_password1" >New Password*</label>
+                            <div class="wrn-msg text-highlight" ng-messages="changepwdForm.new_password1.$error" ng-if="changepwdForm.new_password1.$touched || changepwdForm.$submitted">
+                                <p ng-message="minlength">password is less than 8 characters.</p>
+                                <p ng-message="required">New password is required</p>
+                            </div>
                         </div>
 
                         <div class="input-field align-left">
-                            <input id="new_password2" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password2" focus-if>
-                            <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
-                            <label for="new_password2" data-error="wrong" data-success="right">New Password (Again)*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.new_password2">{{changepwd.wrnMsg.new_password2}}</div>
+                            <input id="new_password2" name="new_password2" type="password"  text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password2" ng-minlength="8" compare-to="changepwd.user.new_password1" required >
```

Comment:
You had not to delete class.

before: `class="validate text-dark-black dark-autofill w-400"`
after : `class="text-dark-black dark-autofill w-400"`